package.json

```diff
@@ -6,6 +6,8 @@
   "version": "0.0.0",
```

Comment:
Can you please add bounds for the stuff that you really want to bump?

Reply:
hmm i think this one is just a whitespace change that came along for the ride : i'll try to avoid touching this file

Reply:
or maybe i didn't understand your question?  are you saying to specifying range rather than specific versions?  or what exactly is the request?

Reply:
Sorry, I wasn’t commenting on this specific line.
I’m saying that we should bump lower bounds for things like `websocket-extensions` if we want to avoid a version with security issues instead of relying on what you got in the `yarn.lock` today. Otherwise you might accidentally introduce the vulnerability again next time you generate the yarn.lock.

Reply:
ah right yeah i didn't touch websocket-extensions as i think that was the sensitive one with manual navigator dev testing where a blind uptick had some implications, but i'm not 100% sure of how to verify the dev setup

i think let's merge this PR if no objections and then see what warnings this clears and we can do a follow-on for others.  does that work?

Reply:
Sounds reasonable :+1: